game/screen_paths.cc

```diff
@@ -101,11 +101,10 @@ void ScreenPaths::generateMenuFromPath(fs::path path) {
 				std::clog << "screen_paths/notice: Ignoring hidden folder: ." << p.c_str() << std::endl;
 				continue;
 			} else {
-				m_menu.add(MenuOption(p.c_str(),_("Open folder")).call([this, p](){
-					generateMenuFromPath(p);
+				m_menu.add(MenuOption(boost::lexical_cast<std::string>(p),_("Open folder")).call([this, p](){ generateMenuFromPath(p);
```

Comment:
would be nice to see the error/stackstrace of the windows build. Probably due to the fact `c_str()` vs `std::string`

Oh btw instead of doing `boost::lexical_cast` one can just run `p.string()`